src/pages/components/tile/usage.mdx

```diff
@@ -598,28 +583,25 @@ base tiles.
 
 ![Base tiles are static, except buttons or links .](images/tile-usage-25.png)
 
-<Caption>Base tiles are static, except buttons or links .</Caption>
+<Caption>Base tiles are static, except buttons or links</Caption>
 
 </Column>
 </Row>
 
 Expandable tiles can either be triggered as a whole or only contain internal
-CTA's if they are given their own click targets and the tile’s click target is
-reduced to only the chevron icon.
-
-<InlineNotification>
-
-**Note:** Clickable targets are currently not available in coding. Teams can
-override the code to make smaller click targets while we're updating this in the
-future.
-
-</InlineNotification>
+CTAs if they are given their click targets and the tile’s click target is
+reduced to only the icon button.
 
 <Row>
 <Column colLg={8}>
 
 ![Clickable, expandable, and selectable tiles are clickable anywhere within the tile.](images/tile-usage-26.png)
 
+<Caption>
+  Expandable tiles can either be triggered as a whole or the click target is
```

Comment:
Expandable tiles can either be triggered as a whole or contain internal CTAs only if they are given their own click targets. In these cases, the tile’s click target is reduced to only the icon button.